assets/svg/index.tsx

```diff
@@ -0,0 +1,4 @@
+import Eye from './eye'
+import EyeSlashed from './eye-slashed'
+
+export { Eye, EyeSlashed }
```

Comment:
What's the purpose of this?

Reply:
เวลามี svg หลายๆอันใน folder ก็จะได้ import จาก file เดียวกัน แต่ตอนนี้จะไม่ใช้ svg แล้วเพราะใช้ MaterialUI icon แทน